src/cli.js

```diff
@@ -0,0 +1,97 @@
+#!/usr/bin/env node
+//TODO: add a .commentsignore file for files which you would like to ignore from the --list.
+/*
+ * c
+ * https://github.com/rumpl/c
+ *
+ * Copyright (c) 2012 Djordje Lukic
+ * Licensed under the MIT license.
+ */
+
+"use strict";
+var commands = require("./commands");
```

Comment:
All the `require`s should be `const`

Reply:
Does this also apply to exports?

Reply:
What exports are we talking about?

Reply:
For instance in storage, line 16: `var storage = module.exports` could become `const storage`?